test/test-ExperimentProvider.js

```diff
@@ -36,5 +38,32 @@ exports["test ActivityStreams doesn't init experimentProvider"] = assert => {
   assert.deepEqual(as._experimentProvider.data, {}, "should have empty .data");
   as.unload();
 };
+exports["test ActivityStreams doesn't init experimentProvider when pref is off"] = assert => {
```

Comment:
Probably describe the test as making sure the user is in the control group as opposed to just being "doesn't init."